frontend-admin/src/app/components/mnoe-api/admin/current-user.svc.coffee

```diff
@@ -17,7 +17,7 @@
   # Save the current user in variable to be able to reference it directly
   @user = {}
 
-  # Get the current user admmin role
+  # Get the current user admin role
   @getAdminRole = ->
```

Comment:
While we're at it:
- Can you rename this function `getUser()`
- Assign (or `angular.copy` if needed) the response to `_self.user` directly, not only the role